chirp/drivers/tdh8.py

```diff
@@ -1477,12 +1478,17 @@ def _get_settings(self):
                               RadioSettingValueList(
                                   SYNC_LIST,
                                   current_index=_settings.sync))
+
             basic.append(rs)
 
-            rs = RadioSetting("save", "Battery Save",
+            rs = RadioSetting("save",
+                              "Power Save" if self.MODEL in H3_LIST
+                              else "Battery Save",
                               RadioSettingValueList(
-                                  BTV_SAVER_LIST,
+                                  H3_BTV_SAVER_LIST if self.MODEL in H3_LIST
+                                  else BTV_SAVER_LIST,
```

Comment:
This is pretty dense, so it'd be better if you just set the list on the class (once for H8, override for H3) and reference `self.SAVER_LIST` here or something. Anyway, not a huge deal so I'll let it slide, but if you want to follow up with a cleanup that would be appreciated :)